catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -314,6 +314,10 @@ public AttributeFormat getAttributeFormat() {
                 false /* tokenized */, false /* multivalued */, GEO_TYPE));
         descriptors.add(new AttributeDescriptorImpl(Metacard.DESCRIPTION, true /* indexed */, true /* stored */,
                 false /* tokenized */, false /* multivalued */, STRING_TYPE));
```

Comment:
Constants here - either move them to Metacard or use the ones you define in the MetacardValidityPlugin